src/client/auth/authActions.js

```diff
@@ -111,6 +112,12 @@ export const login = (oAuthToken = '', socialNetwork = '', regData = '') => asyn
 
   const isGuest = waivioAPI.isGuest;
 
+  const platformName = get(state, ['platform', 'platformName'], '');
+  const userName = get(state, ['auth', 'user', 'name'], '');
+
+  const isBeaxyUser = userName.split('_')[0] === 'bxy';
```

Comment:
create helper function isBeaxyUser(username). Use it here and in BrokerBalance component

Reply:
++